apps/editor.planx.uk/src/pages/FlowEditor/lib/store/shared.ts

```diff
@@ -91,6 +91,7 @@ export const sharedStore: StateCreator<
       changedNode: undefined,
       saveToEmail: "",
       currentCard: null,
+      hasAcknowledgedSingleSessionEntry: false,
```

Comment:
On "restart", we nullify the `saveToEmail` for submission services and now set `hasAcknowledgedSingleSessionEntry` to false - both of these help control visibility of the default entry page for their respective service "types".

Is this still a reasonable responsibility of the `shared` store?